test/SecTester.Scan.Tests/Extensions/TaskExtensions.cs

```diff
@@ -0,0 +1,9 @@
+namespace SecTester.Scan.Tests.Extensions;
+
+public static class TaskExtensions
+{
+  public static TResult? GetSync<T, TResult>(this T? t, Func<T, Task<TResult>> func)
```

Comment:
Please follow conventions

```suggestion
  public static TResult? ContinueWithSycn<T, TResult>(this T? t, Func<T, Task<TResult>> continuationFunction)
```